.github/workflows/translations-sync.yml

```diff
@@ -35,48 +35,46 @@ jobs:
+      # Downloads translations from Crowdin and creates a PR
+      # See all the options at https://github.com/crowdin/github-action
+      - name: Download Translations & Create PR
         uses: crowdin/github-action@b8012bd5491b8aa8578b73ab5b5f5e7c94aaa6e2 # v2.7.0
-        id: crowdin_pr
         with:
-          # do not upload anything - this is a one-way operation download
           upload_sources: false
           upload_translations: false
-          # the rest of this controls how the PR comes in with new translations
           download_translations: true
-          localization_branch_name: chore/crowdin
+          localization_branch_name: ${{ env.HEAD_REF }}
           create_pull_request: true
-          pull_request_title: '[automated]: crowdin sync'
-          pull_request_body: 'New Crowdin translations from the [Node.js Crowdin project](https://crowdin.com/project/nodejs-web)'
-          commit_message: 'chore: synced translations from crowdin'
+          pull_request_title: '[automated]: Crowdin Translations Sync'
+          pull_request_body: 'New translations from the [Node.js Crowdin project](https://crowdin.com/project/nodejs-web)'
+          commit_message: ${{ env.COMMIT_MSG }}
         env:
           GITHUB_TOKEN: ${{ secrets.CROWDIN_GITHUB_BOT_TOKEN }}
-          # A numeric ID, found at https://crowdin.com/project/nodejs-web/tools/api
           CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
-          # Created from https://crowdin.com/settings#api-key logged in using nodejs-crowdin-bot
           CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
 
-  format_crowdin_pull_request:
-    needs: synchronize-with-crowdin
+  format-translations:
+    needs: sync-translations
     runs-on: ubuntu-latest
-
-    permissions:
-      # This permission is required by `stefanzweifel/git-auto-commit-action`
-      contents: write
-
     steps:
       - name: Harden Runner
-        uses: step-security/harden-runner@0634a2670c59f64b4a01f0f96f84700a4088b9f0 # v2.12.0
+        uses: step-security/harden-runner@0634a2670c59f64b4a01f0f96f84700a4088b9f0
         with:
           egress-policy: audit
 
-      - name: Git Checkout
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      # Checks out the PR branch created by the previous job
+      - name: Checkout PR Branch
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
         with:
-          # Use the number from the output of crowdin/github-action
-          ref: refs/pull/${{ needs.synchronize-with-crowdin.outputs.pull_request_number }}/head
+          ref: ${{ env.HEAD_REF }}
           token: ${{ secrets.CROWDIN_GITHUB_BOT_TOKEN }}
+          fetch-depth: 0
+
+      # Setup git user for commits
+      - name: Setup Git User
+        run: |
+          git config --global user.name "nodejs-crowdin"
+          git config --global user.email "nodejs-crowdin@users.noreply.github.com"
```

Comment:
unknown if this is better than the default behavior provided by stefanzweifel/git-auto-commit-action